main.js

```diff
@@ -78,18 +78,8 @@ async function run() {
   })
     .then(async generatedSwaggerString => {
       const oas = new OAS(generatedSwaggerString);
-      const validate = promisify(oas.validate);
```

Comment:
Can we remove any dependencies now? Or is this all inside of the oas-normalize module?

Reply:
Yeah we still use the `oas-normalize` package for this bundling: https://github.com/readmeio/github-readme-sync/blob/dd9110dfd7f05f215f9cbc8a3e81a5ea703b94cd/main.js#L81